cli/cmd/runtime/start.go

```diff
@@ -91,6 +91,7 @@ type Config struct {
 	ActivitySinkKafkaBrokers string `default:"" split_words:"true"`
 	// Kafka topic of an activity client's sink
 	ActivitySinkKafkaTopic string `default:"" split_words:"true"`
+	EmitInstanceHeartbeat  bool   `default:"false" split_words:"true"`
```

Comment:
Let's not make this configurable at this stage. You can already disable all metrics (by setting`ActivitySinkType` to noop as we do in dev), and I don't think we need more granular control than that at this stage.